app/models/manageiq/providers/autosde/inventory/persister/storage_manager.rb

```diff
@@ -1,29 +1,43 @@
 # This class builds the collections. The parser then adds actual inventory items to the collections.
 # Generally, one collection per application-record class.
+# for more info see: https://www.manageiq.org/docs/guides/providers/persister/inventory_collections
 class ManageIQ::Providers::Autosde::Inventory::Persister::StorageManager < ManageIQ::Providers::Inventory::Persister
   # @return [Hash{Symbol => InventoryRefresh::InventoryCollection}] collections
   attr_reader  :collections
 
   def initialize_inventory_collections
-    # physical storages
-    add_collection(storage, :physical_storages) do |builder|
-      builder.add_default_values(:ems_id => ->(persister) { persister.manager.id })
-      builder.add_properties(
-        :model_class => ManageIQ::Providers::Autosde::StorageManager::PhysicalStorage
-      )
-    end
+    collect_san_addresses
```

Comment:
`collect_` is a bit confusing because this is in the persister not the collector.  If you have provider overrides the typical pattern is to call this `add_host_initiators`